### PR TITLE
859 weapons item sheet and card upgrade

### PR DESCRIPTION
**Pull request by patrickmohrmann**
a lot of documentation is part of this issue which is related to Physical items. The documentation has no direct link to the smaller changes i made, I just forgot to switch the branch and don't know /want to (because i fear everything will break :-) ) try to split it.

### CODE REVIEW

File: module/data/item/templates/physical-item.mjs [hunk 1]
```diff
@@ -109,6 +109,10 @@ export default class PhysicalItemTemplate extends SystemDataModel.mixin(
                       integer: true,
                       label: "ED.Item.General.arbitraryStep"
                   } ),
+                  action: new fields.StringField( {
```
Comment (ChristopherSD) [inline module/data/item/templates/physical-item.mjs:112]: This should use `ED4E.action` as `choices`

File: module/data/item/equipment.mjs [hunk 2]
```diff
@@ -19,12 +19,14 @@ export default class EquipmentData extends PhysicalItemTemplate.mixin(
                 label: "ED.Item.Equipment.consumable"
             } ),
             // different ammo types are availabel see issue #
-            ammoType: new fields.StringField( {
+            ammunition: new fields.SchemaField( {
```
Comment (ChristopherSD) [inline module/data/item/equipment.mjs:22]: I can still find 3 occurrences of "ammoType" on this branch... They should be corrected

File: module/documents/actor.mjs [hunk 3]
```diff
@@ -236,6 +237,13 @@ export default class ActorEd extends Actor {
     this.#processRoll( roll );
   }
 
+  /**
+   * 
+   * @param {Object}    itemId        Id of the item to rotate the status of
```
Comment (ChristopherSD) [inline module/documents/actor.mjs:242]: Should probably have a description of the sequence that is rotated (I didn't document it, oops....)

File: module/handlebar-helpers.mjs [hunk 4]
```diff
@@ -97,4 +96,46 @@ export default function registerHandlebarHelpers() {
...
+  /*            Handlebars for Item sheets             */
+  /*************************************************** */
+  Handlebars.registerHelper("isRangedWeapon", function(weaponType) {
+    // Define the logic to determine if the weapon is ranged
+    const rangedWeaponTypes = ["bow", "crossbow", "blowgun", "thrown"]; // Example types
+    return rangedWeaponTypes.includes(weaponType);
+  })
+
+  /*************************************************** */
+  /*            Handlebars for Item Cards              */
```
Comment (ChristopherSD) [inline module/handlebar-helpers.mjs:111]: These functions are all used only once in a template and way to specific, "single-use". So they probably shouldn't be helpers but just be added in the corresponding `prepareData` methods (this should generally be the first choice, unless it is something very general and often used)

File: lang/de.json [hunk 5]
```diff
@@ -125,6 +125,7 @@
                 "action":                                           "Aktion",
                 "active":                                           "Aktiv",
                 "amount":                                           "Menge",
+                "ammunition":                                       "Munition / Menge",
```
Comment (ChristopherSD) [inline lang/de.json:128]: It says `ammunition.type` in the localization calls

**Review by ChristopherSD — CHANGES_REQUESTED** [review]

File: module/data/item/templates/physical-item.mjs (hunk 1 above)
Comment (patrickmohrmann) [reply module/data/item/templates/physical-item.mjs:112]: changed it together with the hbs.

File: lang/de.json (hunk 5 above)
Comment (patrickmohrmann) [reply lang/de.json:128]: ok fixed the relevant parts

File: module/data/item/equipment.mjs (hunk 2 above)
Comment (patrickmohrmann) [reply module/data/item/equipment.mjs:22]: fixed it

File: module/handlebar-helpers.mjs (hunk 4 above)
Comment (patrickmohrmann) [reply module/handlebar-helpers.mjs:108]: removed handlebars handler and included them in the weapons dataModel as getter and the base-sheet for getData

**Review by ChristopherSD — APPROVED** [review]

**Pull request merged**